src/helpers/blocks.js

```diff
@@ -0,0 +1,36 @@
+import utils from "ethereumjs-util"
+
+const sha3 = utils.sha3
+const BN = utils.BN
+
+export async function getHeaders(start, end, web3) {
+  if (start >= end) {
+    return []
+  }
+
+  let current = start
```

Comment:
if we don't have any plan for getting current variable's value from somewhere else then using start variable seems ok.

Reply:
@0xAshish context?

Reply:
Minor thing : `let current = start` is not needed assignment, instead use `start` itself.

Reply:
True